wagtail/contrib/forms/models.py

```diff
@@ -152,10 +152,8 @@ class Meta:
         ordering = ["sort_order"]
 
 
-class AbstractForm(Page):
-    """
-    A Form Page. Pages implementing a form should inherit from it
-    """
+class FormPageMixin:
```

Comment:
```suggestion
class FormMixin:
```

Minor change - going with `FormMixin` here makes more sense as we do not use the term `Page` elsewhere in the abstract models or mixins.